examples/simple_viewer/index.html

```diff
@@ -0,0 +1,517 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+
+    <title>DICOM WSI Viewer</title>
+    <meta charset="utf-8" />
+    <link type="text/css" rel="stylesheet" href="styles/main.css">
+    <script type="text/javascript" src="https://unpkg.com/dicomweb-client"></script>
```

Comment:
@swederik said he was having trouble with unpkg refreshing unless he included the version.  See:  https://github.com/dcmjs-org/dcmjs-examples/commit/5367df2ba36131ee8f7f9df9de054b58da89ba3b

Reply:
Addressed with 7789763f1edee6a1a49353d70951488c6478a276